discretisedfield/region.py

```diff
@@ -286,7 +305,7 @@ def random_point(self):
            ``discretisedfield.Region.random_point()`` method is called.
 
         """
-        return dfu.array2tuple(np.random.random(3) * self.edges + self.pmin)
+        return np.random.random(3) * self.edges + self.pmin
 
     def __repr__(self):
         r"""Representation string.
```

Comment:
@lang-m how do you think is best to go about this because `type(region.pmin)=np.ndarray` and we currently only pass region to jinja2 so would we have to have a property of region which is a tuple of pmin?

Reply:
I will think about __repr__ and _repr_html_ and fix it when everything else is done. Please just ignore it for now.